### DISCUSSION TITLE
How to configure quarkus app on gke to use a sidecar proxy

### DISCUSSION THREAD

**Discussion by Endagegnehu** [quarkusio/quarkus#32550] [Q&A]
Hello, new to Quarkus; how do I connect to a cloud SQL sidecar proxy that is running on GKE?

**Comment by geoand** [comment]
cc @loicmathieu @patriot1burke

**Comment by loicmathieu** [comment]
Hello,
You can find information on how to use Cloud SQL with Quarkus here: https://quarkus.io/guides/deploying-to-google-cloud#using-cloud-sql

In the documentation page of the Cloud SQL Auth Proxy there is a note that said it's not designed to be used from Java so I think the best is to use the way documented on the Quarkus guide above.

> If you're using the Java, Python, or Go languages, then [connect with the corresponding connector](https://cloud.google.com/sql/docs/mysql/connect-connectors) instead of with the Cloud SQL Auth proxy.

Source: https://cloud.google.com/sql/docs/mysql/sql-proxy
